config.mk

```diff
@@ -0,0 +1,6 @@
+DOCKER_ROOT_IMAGE := ubuntu:18.04
```

Comment:
This is not (yet) used but we should push it through so it is easy to upgrade the root image.